packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -66,14 +73,16 @@ class TabPanel extends React.Component {
 
     return (
       <ThemeProvider theme={theme}>
-        <div>
+        <div className={className}>
           <AriaAnnouncer id="announcer" aria-live="assertive">{`${
             this.state.simpleName
           } Sub text is now showing`}</AriaAnnouncer>
-          <TabWrapper>
-            <TabFormatter>{elements}</TabFormatter>
+          <TabWrapper className="tab-wrapper">
```

Comment:
As we add these identifying classes to the components, I'd like to prefix them with "es-" so they are easily set apart from other classes.